CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+#Contributing
+
+Thank you for your interest in contributing to Eventbrite's Javascript SDK!
+
+###Table of Contents
+* [Workflow](#workflow)
+  * [Setup](#setup)
+  * [Branches](#using-branches-to-submit-changes)
+  * [Keeping up to date](#keeping-your-local-repo-up-to-date)
+* [Creating issues](#creating-issues)
+* [Working on and submitting changes](#working-on-and-submitting-changes)
+  * [Steps to submit](#steps-to-submit)
+
+## Workflow
+
+### Setup
+
+1. Fork the repository
```

Comment:
Can you link "Fork the repository" to: https://help.github.com/articles/fork-a-repo/